### PR TITLE
updated vsphere getting started guide

### PR DESCRIPTION
**Pull request by divyenpatel**
updated vsphere getting started guide - Known issues section.

cc: @kerneltime @abrarshivani @BaluDontu @tusharnt @pdhamdhere

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2570)

### CODE REVIEW

File: docs/getting-started-guides/vsphere.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Sample Config:
 
 #### Known issues
 
-* [Volumes are not removed from a VM configuration if the VM is down](https://github.com/kubernetes/kubernetes/issues/33061). The workaround is to manually remove the disk from VM settings before powering it up.
+* [Volumes are not removed from a VM configuration if the VM is down](https://github.com/kubernetes/kubernetes/issues/33061). The workaround is to manually remove the disk from VM settings before powering it up. - This issue is fixed in 1.5.3
```
Comment (pdhamdhere) [inline docs/getting-started-guides/vsphere.md]: Both known issues are fixed in 1.5.3. Right? I would suggest to simply remove the section.

And add "Recommended Kubernetes Version: 1.5.3" under "vSphere Cloud Provider" section.

Comment (BaluDontu) [inline docs/getting-started-guides/vsphere.md]: Yes please remove this bug since it is already fixed.

Comment (divyenpatel) [inline docs/getting-started-guides/vsphere.md]: removed. added issue regarding kubectl exec - https://github.com/kubernetes/kubernetes-anywhere/issues/337

File: docs/getting-started-guides/vsphere.md [hunk 2]
```diff
@@ -79,8 +79,7 @@ Sample Config:
 
 #### Known issues
 
-* [Volumes are not removed from a VM configuration if the VM is down](https://github.com/kubernetes/kubernetes/issues/33061). The workaround is to manually remove the disk from VM settings before powering it up.
-* [FS groups are not supported in 1.4.7](https://github.com/kubernetes/kubernetes/issues/34039) - This issue is fixed in 1.4.8
+* [Unable to execute command on pod container using kubectl exec] (https://github.com/kubernetes/kubernetes-anywhere/issues/337)
```
Comment (chenopis) [inline docs/getting-started-guides/vsphere.md]: Typo causing link not to be rendered correctly: remove space between ']' and '('

Comment (divyenpatel) [inline docs/getting-started-guides/vsphere.md]: Thanks @chenopis fixed it.

**Pull request merged**